CHANGELOG-UNRELEASED.md

```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 - Added App Validation workflow that runs app validation as authority [#330](https://github.com/holochain/holochain/pull/330)
 - BREAKING: get_details and get_links_details return SignedHeaderHashed instead of the header types [390](https://github.com/holochain/holochain/pull/390)
+- BREAKING: ZomeInfo now returns the ZomeId [390](https://github.com/holochain/holochain/pull/390)
```

Comment:
Is this considered "breaking" though its a new addition?

Reply:
If someone is destructing the struct it will break their code